src/_guides/technical-writing/5-improvements-to-openapi-operation-documentation.md

```diff
@@ -0,0 +1,255 @@
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+```
+
+### Improved Version with Example Values:
+Adding examples helps users understand how to interact with the API more effectively. It illustrates what to expect in requests and responses.
+
+**Improved Example:**
+```yaml
+components:
+  schemas:
+    User:
+      type: object
+      properties:
+        id:
+          type: string
+          example: "a3a39bcb1f94049d"
+        name:
+          type: string
+          example: "John Doe"
+```
+
+By leveraging values that are synthetic but perhaps more realistic, it helps the developer to better understand what the operation may require in a request or return in a response. Additionally, maintaining consistency between examples help to improve the understanding of how the API operation works. 
+
+## 3. Add Common Response Code Documentation
+
+With limited time available, API operations may only include documentation for common response codes like `200 OK` and `500 Internal Server Error`, neglecting other important status codes. Status codes such as `201 Created`, `202 Accepted`, `400 Bad Request`, `401 Unauthorized`, and `404 Not Found` are also useful to document. 
+
+Some API providers have opted to use [RFC 9457 Problem Details](https://www.rfc-editor.org/rfc/rfc9457) format (formally RFC 7807) to provide a consistent error message format. By documenting common response codes, not only does it inform the consumer of what to expect when errors occur, it also guides them on the error format of the response. 
+
+Of course, any HTTP client must be prepared to handle any response code - even those not documented in the OpenAPI documentation for your API. However, the more response codes you explicitly document, the more likely they will catch and handle those errors within the client code. 
+
+### Example Operation with Limited Response Codes
+
+**Example Mistake:**
+```yaml
+responses:
+  200:
+    description: "Success"
+  500:
+    description: "Server Error"
+```
+
+### Improved Version with Common Error Response Codes
+
+**Improved Example:**
+```yaml
+responses:
+  200:
+    description: "Success"
+    …
+  400:
+    description: "Bad Request - Invalid parameters"
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/ProblemDetails'
+  401:
+    description: "Unauthorized - Authentication failed"
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/ProblemDetails'
+  404:
+    description: "Not Found - Resource doesn’t exist"
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/ProblemDetails'
+  500:
+    description: "Internal Server Error - System encountered an unexpected error"
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/ProblemDetails'
+```
+
+## 4. Organize and Tag Your API Operations
```

Comment:
Could you add a link to a guide we published (first on our blog, then as a guide here in the content hub) a year ago. Not sure if at the begining or at the end of your paragraph, I'll let you decide!

Here's the guide about organization and tagging operations: https://docs.bump.sh/guides/api-basics/openapi-asyncapi-tags-organize-endpoints/

Reply:
Added to the end as a "refer to..." CTA